notebooks/experimental/semantic_operators.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Nit: Use second-person to direct the reader.</p><p> </p><p>"First, import the BigQuery DataFrames modules."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>